src/_includes/templates/post-details.njk

```diff
@@ -14,7 +14,7 @@
     </time>
   </p>
 
-  <p>{{ readingTime.minutes }} min read</p>
+  <p>{{ readingTime.minutes }} {{ static_text.reading_time|default("min read") }}</p>
```

Comment:
If all copies are included in the _data file (in English by default), we don't need to use the `default()` filter here.